cloud_automation/gcf/go.sum

```diff
@@ -0,0 +1,461 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
the .sum file seems to be too long given the go.mod file only have a few library